boards/av/x-v1/init/rc.board_extras

```diff
@@ -5,7 +5,7 @@
 #
 
 # AV-X: start MAVLink to companion (connected to TX2)
-mavlink start -d /dev/ttyS5 -r 20000 -b 460800 -m onboard
+mavlink start -d /dev/ttyS5 -b 500000 -m osd
```

Comment:
Is this change intentional?

Reply:
Yes, it is. With onboard mode, we had to many data drops, so changed it to osd

Reply:
What about the baudrate? I thought 500000 was problematic for the tx2.